selftests/unit/test_remote.py

```diff
@@ -35,10 +35,10 @@ def setUp(self):
                         remote_port=22,
                         remote_password='password',
                         remote_key_file=None,
-                        remote_no_copy=False,
                         remote_timeout=60,
                         show_job_log=False,
-                        mux_yaml=['foo.yaml', 'bar/baz.yaml'],
+                        mux_yaml=['~/avocado/tests/foo.yaml',
+                                  '~/avocado/tests/bar/baz.yaml'],
```

Comment:
yes, but still when the fabric connects to the remote location, it is in a certain directory and you can specify relative path from there (eg. the `foo.yaml`).